modmesh/app/euler1d.py

```diff
@@ -743,24 +743,35 @@ def update_lines(self):
         :return: None
         """
         if self.use_grid_layout:
-            self.density.update(adata=self.st.density_field,
-                                ndata=self.st.svr.density[::2])
-            self.pressure.update(adata=self.st.pressure_field,
-                                 ndata=self.st.svr.pressure[::2])
-            self.velocity.update(adata=self.st.velocity_field,
-                                 ndata=self.st.svr.velocity[::2])
-            self.temperature.update(adata=self.st.temperature_field,
-                                    ndata=self.st.svr.temperature[::2])
-            self.internal_energy.update(adata=(self.st.internal_energy_field),
-                                        ndata=(self.st.svr.
-                                               internal_energy[::2]))
-            self.entropy.update(adata=self.st.entropy_field,
-                                ndata=self.st.svr.entropy[::2])
+            self.density.update(
```

Comment:
Please simplify the code like:

```python
            _s = self.st.svr.xindices
            self.density.update(adata=self.st.density_field,
                                ndata=self.st.svr.density[_s])
```

In this way, you do not change the original code by too much.  It will be easier to keep track of the change.